### PR TITLE
174 live page

### PR DESCRIPTION
**Pull request by julianahhong**
created day-of page with cards layout. still need to include hacker earth link

### CODE REVIEW

**Comment by TrevorEdwards** [comment]
Can you make the boxes for the counters constant-width? The seconds box changes with digit count a bit

**Comment by TrevorEdwards** [comment]
It's also a bit lacking in color in spots. Maybe we can make the grey boxes light blue?

**Comment by TrevorEdwards** [comment]
also make button colors the same (submit/request mentor)

**Comment by TrevorEdwards** [comment]
twitter feed only gives a link to twitter for my machine.. not sure why

**Comment by TrevorEdwards** [comment]
Aside from those, I think this looks a lot nice than what we had before!

File: views/live.jade [hunk 1]
```diff
@@ -1,29 +1,123 @@
...
+        .row
+            .col-md-4
+                .row
+                    .panel.panel-default
+                        .panel-heading
+                            h3.panel-title Info
+                        .panel.panel-default
+                            .wifi
+                                h4 wifi: Cornell-Visitor
+                                h4 hacking ends: 9/18, 10 am
```
Comment (TrevorEdwards) [inline views/live.jade]: 8 am

File: routes/api/admin.js [hunk 2]
```diff
@@ -1010,6 +1010,7 @@ function setInventory(req, res, next) {
  * @apiParam {Number} quantity The quantity of hardware to transact
  * @apiParam {String} name The unique name of the hardware being transacted
  **/
+
```
Comment (mahsu) [inline routes/api/admin.js]: remove

File: routes/middleware.js [hunk 3]
```diff
@@ -48,7 +48,7 @@ middle.requireAdmin = function (req, res, next) {
 };
 
 middle.requireMentor = function (req, res, next) {
-    if (req.user && req.user.role === "mentor") {
+    if (true) { // TODO
```
Comment (mahsu) [inline routes/middleware.js]: Is this fixed in the mentorship pr?

File: views/live.jade [hunk 4]
```diff
@@ -1,29 +1,123 @@
...
+                                    tr
+                                        .message
+                                            td #{announcement.message}
+                                        .time
+                                            td.time #{moment(announcement.time).from()}
+                .row
+                    .panel.panel-default
+                        .panel-heading
+                            h3.panel-title HackerEarth
+                                //-to do- change link here
```
Comment (mahsu) [inline views/live.jade]: make an issue and reference it here

File: views/live.jade [hunk 5]
```diff
@@ -1,29 +1,123 @@
...
 
+                .row
+                    .panel.panel-default
+                        .panel-heading
+                            h3.panel-title Schedule
+                        .calendar
+                            .schedule-tab
+                                .heading
+                                    h3 fri, 9/16
+                                for event in calendar
```
Comment (mahsu) [inline views/live.jade]: is the schedule only showing events in the future?

File: views/mentor/index.jade [hunk 6]
```diff
@@ -1,47 +1,75 @@
...
+        | Claimed:
+    |  A request that has been claimed, and the mentor who claimed it will be shown.
+    br
+    span.completed
+        | Completed:
+    |  A request that has been fully completed after a mentor claimed it and then resolved the student's request.
+    br
+    .header-wrapper
+        h2.page-title requests
+    //if mentorRequests.length == 0
```
Comment (mahsu) [inline views/mentor/index.jade]: remove commented

File: views/mentor/index.jade [hunk 7]
```diff
@@ -1,47 +1,75 @@
...
+            | skill set in dashboard home to increase the number of requests that come your way.
+    //else
+        #usermentorrequests
+            h5
+                input#onlyunclaimed(type='checkbox', name='onlyunclaimed')
+                |  show only unclaimed requests
+            h5
+                input#onlymatching(type='checkbox', name='onlymatching')
+                |  show only matching requests
+    //- for request in mentorRequests
```
Comment (mahsu) [inline views/mentor/index.jade]: remove commented

File: views/register_mentor.jade [hunk 8]
```diff
@@ -4,23 +4,21 @@
 extends _layout-registration
 
 block vars
-    - companyNames = enums.mentor.companyname;
-    - var actionUrl = "/mentorregistration"; //- url the form should submit to
+    - var actionUrl = "/mentor/register"; //- url the form should submit to
     - var multipart = true; //- whether the form has multipart data
```
Comment (mahsu) [inline views/register_mentor.jade]: it doesnt look like this form has multipart data?

When processing results, does it spit out multipart?

**Comment by mahsu** [comment]
how easy is it to make additional modules? We're going to need to add a link to the program doc and possibly to the mobile apps.

**Comment by mahsu** [comment]
lgtm

**Comment by julianahhong** [comment]
@mahsu should be pretty easy to add things

**Pull request merged**